sdk/core/azure-core-tracing-opentelemetry/inc/azure/core/tracing/opentelemetry/opentelemetry.hpp

```diff
@@ -5,9 +5,6 @@
 
 #include <azure/core/tracing/tracing.hpp>
 
-#if defined(_azure_APIVIEW)
```

Comment:
Looks like there is an instance of `_azure_APIVIEW` here:
https://github.com/Azure/azure-sdk-for-cpp/blob/15d8133a215a1f1d93c40714455e9f4e8afb4013/sdk/core/azure-core-tracing-opentelemetry/inc/ApiViewSettings.json#L9

Could you look through and cleanup all such instances as part of this (maybe GitHub search missed some)?
https://github.com/search?q=repo%3AAzure%2Fazure-sdk-for-cpp%20_azure_APIVIEW&type=code

Reply:
Thanks! I fixed it. This should be the only one missing.
What happened is that "Find" seems to be broken in my Visual Studio (which is why I initiated a PR to add all the files to cmakelists, because I thought that was the reason - it wasn't, but still I kept the PR). I opened in VSCode, and it found that extra instance.